designs/lucy/src/pocket.js

```diff
@@ -51,10 +51,10 @@ export default function (part) {
       .line(points.middle)
       .attr('class', 'path fabric dashed')
 
-    points.logo = points.center.shiftOutwards(points.middle, pocketWidth / 5)
+    points.logo = points.centerLeft.shift(270, pocketWidth / 3)
```

Comment:
I note that the new location can sometimes cause the logo to encroach upon the left seam, for example if the width is decreased. Your current change is an improvement from what exists now so I'm okay with the change as is. However, you might want to consider lowering it a bit more.
![Screen Shot 2022-08-22 at 3 22 12 PM](https://user-images.githubusercontent.com/109869956/186029403-52080642-0d31-471b-82c0-5c51bfb0919a.png)

Reply:
I will lower it! Skully hates me, but I want them on there!